packages/gen-ai/bff/README.md

```
@@ -1461,6 +1484,91 @@ cd packages/gen-ai
+### Optional: Deploy Kubernetes MCP Server to Cluster
+
+To register a real Kubernetes MCP server in the local MLflow MCP registry:
+
+```bash
+# From packages/gen-ai (one-time, requires oc login)
+make deploy-k8s-mcp
+```
+
+This deploys `bff/testdata/kubernetes-mcp-server.yaml` to the `mcp-servers` namespace and prints:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)